pkg/kubelet/kubelet_test.go

```diff
@@ -158,6 +158,7 @@ func newTestKubeletWithImageList(
 	kubelet.recorder = fakeRecorder
 	kubelet.kubeClient = fakeKubeClient
 	kubelet.heartbeatClient = fakeKubeClient.CoreV1()
+	kubelet.lastHeartbeatTime = metav1.Now()
```

Comment:
why was this change needed?

Reply:
just fix test failure

Reply:
getNodeReadyLastHeartbeatTime(cachedNode) returns unset time in this test case.

Reply:
@liggitt pushed an alternative way to fix ut, ptal.